### PR TITLE
Deprecate UTXO-based currencies for new currency plugin

### PR DESCRIPTION
**Pull request by samholmes**
#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [ ] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202393418150156

### CODE REVIEW

**Review by thehobbit85 — APPROVED** [review]
Approved. Don't merge until the the new currency-plugin is merged.

**Comment by samholmes** [comment]
Replaced by https://github.com/EdgeApp/edge-react-gui/pull/3456

**Pull request closed without merging**
